src/form/v2/birth/forms/pages/child.ts

```diff
@@ -381,8 +382,20 @@ export const child = defineFormPage({
         description: 'This is the label for the field',
         id: 'v2.event.birth.action.declare.form.section.child.field.weightAtBirth.label'
       },
+      validation: [
+        {
+          message: {
+            defaultMessage: 'Must be within 0 and 6',
+            description: 'This is the error message for invalid number range',
+            id: 'v2.error.child.weightAtBirth.invalidNumberRange'
+          },
+          validator: or(
+            field('child.weightAtBirth').isBetween(0, 6),
+            field('child.weightAtBirth').isUndefined()
+          )
+        }
+      ],
       configuration: {
-        min: 0,
```

Comment:
![Screenshot from 2025-03-26 11-48-33](https://github.com/user-attachments/assets/548d6646-cedd-45a1-9b04-a27b5ce5c520)
This vaildation is not needed for these fields